### PR TITLE
ci(github-action): added a workflow to auto-approve dependency updates

### PR DESCRIPTION
**Pull request by travi**

### CODE REVIEW

**Comment by travi** [comment]
i keep losing track of the dependency updates here since the auto-merge is blocked by the need for approval. this workflow should work around that, but only for dependency updates

**Review by micleners — APPROVED** [review]

**Review by micleners — APPROVED** [review]

**Pull request merged**
